### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by MadSpyxFR**

### CODE REVIEW

File: docs/setting.md [hunk 1]
```diff
@@ -4,8 +4,9 @@
 
 Returns a `Boolean` whether Steam APIs were successfully initialized or not.
 
-Note: When testing this, you need to launch and log in the Steam Client, and
-put `steam_appid.txt` under your app directory.
+Note: When testing this, you need to launch and log in the Steam Client, and 
+create a steam_appid.txt file with your Steam APP ID (or the steamworks example APP ID)
```
Comment (hokein) [inline docs/setting.md]: This line exceeds 80 characters a line.

File: docs/setting.md [hunk 2]
```diff
@@ -99,8 +100,16 @@ var Jimp = require('jimp');
 var friends = greenworks.getFriends(greenworks.FriendFlags.Immediate);
 if (friends.length > 0) {
   var handle = greenworks.getSmallFriendAvatar(friends[0].getRawSteamID());
+  if (!handle) {
+    console.log("The user don't have small avartar");
```
Comment (hokein) [inline docs/setting.md]: How about changing the message to `The use don't set small avatar`, which makes it more align with its comments in steamworks SDK.

btw, could you also add this note (`returns 0 if none set`) to `getSmallFriendAvatar` and `GetMediumFriendAvatar` doc?

File: docs/setting.md [hunk 3]
```diff
@@ -99,8 +100,16 @@ var Jimp = require('jimp');
...
+    return;
+  }
   var image_buffer = greenworks.getImageRGBA(handle);
   var size = greenworks.getImageSize(handle);
+  if (!size.height || !size.width) {
+    console.log("Image corrupted. Please try again")
```
Comment (hokein) [inline docs/setting.md]: missing a trailing `;`.

File: readme.md [hunk 4]
```diff
@@ -223,6 +223,8 @@ npm install --save-dev electron-rebuild
 node_modules/.bin/electron-rebuild
 
 # create a steam_appid.txt file with your Steam App ID
+# replace 123456 by your Steam APP ID 
+# or the steampworks example APP ID
 echo 123456 > steam_appid.txt
```
Comment (hokein) [inline readme.md]: I'd prefer to change this line to `echo <your_app_steam_id> > steam_appid.txt`, and remove the above comments.

**Comment by MadSpyxFR** [comment]
Done

**Comment by hokein** [comment]
Nice! Thanks.

**Pull request merged**